crates/store/re_log_types/src/path/parse_path.rs

```diff
@@ -167,6 +167,18 @@ impl EntityPath {
     pub fn parse_forgiving(input: &str) -> Self {
         let mut warnings = vec![];
 
+        // TODO(#9193): Ideally we'd want to print a warning here, but that
+        //              conflicts with how we construct entity paths in our non-Rust SDKs.
```

Comment:
leaving this code snippet in here would mean that only the Rust sdk warns about it but not the others, right? Is that really a problem though?

Reply:
The others would always warn (at least C++).

Reply:
then I didn't understand the problem here

Reply:
So here is my reasoning:

* The C++ API does not expose an `EntityPath` class, and all entity path objects are created from `std::string_view`.
* The conversion from `std::string_view` uses the code path that does the warning.
* In Rust we use `EntityPath::partition_properties` which internally uses `from(parts)`, skipping the code path with the warnings.

The gist is: There is no way to create an `EntityPath` in C++ (and probably Python) without going through the warning code path. This means that every time a user logs recording properties they will see this warning, even though they go through our sanctioned methods.